### PR TITLE
Gamma distribution

### LINKED ISSUE #14: Gamma Distribution

### PR DESCRIPTION
**Pull request by sobir-git**

### CODE REVIEW

**Review by sergey-jr — COMMENTED** [review]
pros: 
- some interactives
- formulas explained briefly

cons: 
- whole material seems too short
- no link to colab to check how interactive works
- few references (as I think)
- images not working well

All seem like "OK", but if you can change something - do it.

**Review by utihamartiwi — COMMENTED** [review]
Good explanation about the distribution. For gamma. in properties part, please give space between theta and variance symbol because the first time I saw it I read like : 
mu = k(theta)(sigma^2)=k(theta^2), so I thought mu= k(theta^2)

**Comment by nikitml** [comment]
@utihamartiwi if everything's fine, please set "approve"
@jacked97  please review
@sobir-git please check comments

**Review by jacked97 — APPROVED** [review]

**Comment by sobir-git** [comment]
Formulas and images are fixed
Added colab link

**Review by sergey-jr — CHANGES_REQUESTED** [review]
pros:

- some interactives
- formulas and material explained briefly
- link to colab
cons:
- some characters are broken

File: gamma-distribution/Gamma_Distribution.ipynb [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "inaYUrnscFqm",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Gamma Distribution - Definition\n",
+        "Gamma distribution is a distribution that arises naturally in processes for which the waiting times between events are relevant. It can be thought of as a waiting time between Poisson distributed events. The exponential distribution, and chi-squared distribution are special cases of the gamma distribution. \n",
+        "Formally, we can parametrize it using shape parameter $k$ and scale $θ$:\n",
```
Comment (sergey-jr) [inline gamma-distribution/Gamma_Distribution.ipynb]: Here is a broken character. You can use as you already used ```\\theta```

**Review by sergey-jr — CHANGES_REQUESTED** [review]
Advice:
- add more examples
- add some tasks
- add some self-questions

**Review by sergey-jr — APPROVED** [review]

**Review by utihamartiwi — APPROVED** [review]

**Pull request merged**